lib/req-utils.js

```diff
@@ -4,7 +4,7 @@ export const getAccessTokenFromReq = (
   req,
   errorMsg = `Not authorized to access ${req.url}. Please provide an authorization header or an app_key.`,
 ) => {
-  const authorizationHeader = get(req, 'headers.authorization') || get(req, 'headers.Authorization');
+  const authorizationHeader = get(req, 'req.headers.authorization') || get(req, 'req.headers.Authorization');
```

Comment:
I had an issue with the request object and had to update this

Reply:
Good catch, there was indeed an issue on this one. It should be fixed with https://github.com/opencollective/opencollective-invoices/pull/281